cmsimple/seofuncs.php

```diff
@@ -15,6 +15,16 @@
  * @link      http://cmsimple-xh.org/
  */
 
+/**
+ * SEO functionality
+ *
+ * @integration of the ADC-Core_XH plugin with extended functions (optional)
+ *
+ * @remove empty path segments in an URL
+ * @remove $su from FirstPublicPage
+ *
+ * @since 1.7.3
```

Comment:
Das Voranstellen eines `@` soll nur bestimmte Tags markieren (in diesem Fall nur `@since`); außerdem dürfte es sinnvoll sein, zu dokumentieren, dass die function nichts zurück gibt, also etwa:
```suggestion
 * Integration of the ADC-Core_XH plugin with extended functions (optional)
 *
 * Remove empty path segments in an URL
 * Remove $su from FirstPublicPage
 *
 * @return void
 *
 * @since 1.7.3
```